### PR TITLE
Post-Training Quantization of MobileNet v2 ONNX Model

### PR DESCRIPTION
**Pull request by alexsu52**
### Changes

Introducing the Post-Training Quantization of MobileNet v2 ONNX model example

### Reason for changes

Moving the example from the experimental module

### Related tickets

97478

### Tests

TBD

### CODE REVIEW

File: examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+onnx
```
Comment (kshpv) [inline examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt:1]: onnx and onnxruntime-openvino are already included in requirments.txt of ONNX backend. Should we expose them here?

**Review by kshpv — CHANGES_REQUESTED** [review]

File: examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt (hunk 1 above)
Comment (alexsu52) [reply examples/post_training_quantization/onnx/mobilenet_v2/requirements.txt:1]: I think, the example has to work if the user installed nncf one of the following ways:
```
pip install nncf
```
```
pip install nncf[onnx]
```
Thus I added onnx and onnxruntime-openvino as dependencies.

Also, I fixed versions of the onnx and onnxruntime-openvino taking them from setup.py.

**Review by kshpv — APPROVED** [review]

**Comment by alexsu52** [comment]
run pylint pre-commit tests

**Comment by aiarkinx** [comment]
run build

**Comment by aiarkinx** [comment]
run onnx pre-commit tests

**Pull request merged**